pkg/keystore/keystore.go

```diff
@@ -9,9 +9,16 @@ import (
 	"errors"
 )
 
+// ErrInvalidPassword is returned when the password for decrypting content where
+// private key is stored is not valid.
 var ErrInvalidPassword = errors.New("invalid password")
 
+// Service for managing keystore private keys.
 type Service interface {
+	// Key returns private key for specified name that was encrypted with
+	// provided password. If the private key does not exists it creates new one
+	// with name and password, and returns with created set to true.
```

Comment:
// Key returns the private key for a  specified name that was encrypted with the
	// provided password. If the private key does not exists it creates a  new one
	// with a name and the password, and returns with `created` set to true.